app/helpers/validation_helper.rb

```diff
@@ -89,6 +89,20 @@ module ValidationHelper # rubocop:todo Metrics/ModuleLength
     'Other'
   ].freeze
 
+  # Please note, this array is only used for the demo.rake file.
+  # The FollowUpFlag component's flag reasons are populated in-line.
+  FOLLOW_UP_FLAG_REASONS = [
```

Comment:
sorry I should I have been more specific, can we put this under `VALID_PATIENT_ENUMS[:follow_up_reason]` for consistency?

Reply:
Putting it in VALID_PATIENT_ENUMS doesn't make a ton of sense to me unless we're using it to run a validation on patients, which I don't see that we are currently.

Not validating follow_up_reason may not necessarily be an issue now since we control how it can be set in the UI; however, I could see it being an issue in the future when it's added to imports and exports and there is a chance of typos, inputting of unsupported reasons, etc. If the follow_up_reason is set to something not in the expected list, then the edit modal on the patient show page appears to break.

https://user-images.githubusercontent.com/24628687/121689916-73a9d000-ca82-11eb-8aa8-7b4cc36e1ab0.mov

Reply:
@jkufro Should I have error handling for that then? I know the UI shouldn't allow the user to input any "Reason" that is not in the drop down list.